internal/provider/resource_gitlab_group_project_file_template_test.go

```diff
@@ -0,0 +1,149 @@
+package provider
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/xanzy/go-gitlab"
+	"os"
+	"strconv"
+	"testing"
+)
+
+func TestAccGitlabGroupProjectFileTemplate_basic(t *testing.T) {
+	// Since we do some manual setup in this test, we need to handle the test skip first.
+	if os.Getenv(resource.EnvTfAcc) == "" {
+		t.Skip(fmt.Sprintf("Acceptance tests skipped unless env '%s' set", resource.EnvTfAcc))
+	}
```

Comment:
We have a test helper for that:

```suggestion
	testAccCheck(t)
```

Reply:
I updated this along with refactoring the resource_gitlab_project_test file, which is where I pulled the original logic from.

Reply:
great! Thanks for going the extra mile 🎉